sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/EventGridPublisherClient.java

```diff
@@ -36,15 +35,17 @@ public EventGridServiceVersion getServiceVersion() {
     /**
      * Publishes the given EventGrid events to the given topic or domain.
      * @param events the EventGrid events to publish.
+     * @throws NullPointerException if events is {@code null}.
```

Comment:
All service methods (on how to use the API), client builder and clients (how to instantiate a client) should have code snippets in javadoc.

Reply:
Added to follow-up issue #19014